.vscode/symbols.json

```diff
@@ -0,0 +1 @@
:{"hasNamespace":false,"type":0,"moduleName":"Nested","relativePath":"src/Nested"},"MultipleValidationError":{"hasNamespace":false,"type":0,"moduleName":"MultipleValidationError","relativePath":"src/MultipleValidationError"},"PackageJson":{"hasNamespace":false,"type":1,"moduleName":"build","relativePath":"src/build"},"ConstraintDefinitionError":{"hasNamespace":false,"type":0,"moduleName":"core","relativePath":"src/core"},"ValidationSchemaNotFound":{"hasNamespace":false,"type":0,"moduleName":"core","relativePath":"src/core"},"ToBeValidOptions":{"hasNamespace":false,"type":1,"moduleName":"testUtil","relativePath":"test/unit/testUtil"},"global.jest.Matchers":{"hasNamespace":true,"type":1,"moduleName":"testUtil","relativePath":"test/unit/testUtil"},"global.jest.Expect":{"hasNamespace":true,"type":1,"moduleName":"testUtil","relativePath":"test/unit/testUtil"},"AssertValidationOptions":{"hasNamespace":false,"type":1,"moduleName":"testUtil","relativePath":"test/unit/testUtil"}},"files":{"../tsdv-joi-consumer/index.ts":"2018-05-20T11:33:25.640Z","src/ValidationResult.ts":"2019-04-26T10:09:44.841Z","src/Nested.ts":"2019-05-12T02:14:36.658Z","src/main.ts":"2019-05-12T10:09:16.255Z","src/MultipleValidationError.ts":"2019-05-12T12:26:38.951Z","src/Validate.ts":"2019-05-12T12:27:37.897Z","src/core.ts":"2019-05-12T12:43:25.939Z","src/Validator.ts":"2018-04-03T10:09:38.122Z","src/build.ts":"2019-05-12T12:27:46.584Z","test/unit/metadataShim.ts":"2018-04-03T10:09:38.124Z","src/constraints/func.ts":"2019-05-11T03:09:51.330Z","src/constraints/date.ts":"2019-05-12T10:44:24.234Z","src/constraints/array.ts":"2019-05-11T03:09:51.327Z","src/constraints/any.ts":"2019-05-11T03:09:51.326Z","src/constraints/string.ts":"2019-05-12T12:17:00.493Z","src/constraints/boolean.ts":"2019-05-11T03:09:51.328Z","src/constraints/number.ts":"2019-05-11T03:09:51.330Z","src/constraints/object.ts":"2019-05-07T12:53:47.275Z","test/unit/testUtil.ts":"2019-05-12T12:45:39.205Z","dist/test/unit/nested.d.ts":"2019-05-11T03:42:02.620Z","dist/test/unit/examples.d.ts":"2019-05-11T03:42:02.562Z","dist/test/unit/Validate.d.ts":"2019-05-11T03:42:02.524Z","dist/test/unit/messages.d.ts":"2019-05-11T03:42:02.601Z","dist/test/unit/gotchas.d.ts":"2019-05-11T03:42:02.577Z","dist/test/unit/inheritance.d.ts":"2019-05-11T03:42:02.593Z","dist/test/unit/constraints/date.d.ts":"2019-05-11T03:42:02.657Z","dist/test/unit/constraints/boolean.d.ts":"2019-05-11T03:42:02.645Z","dist/test/unit/constraints/string.d.ts":"2019-05-11T03:42:02.712Z","test/unit/constraints/any.test.ts":"2019-05-12T05:28:32.293Z","test/unit/constraints/string.test.ts":"2019-05-12T12:34:01.905Z","test/unit/constraints/boolean.test.ts":"2019-05-12T12:31:06.338Z","test/unit/constraints/date.test.ts":"2019-05-12T12:33:34.413Z","test/unit/gotchas.test.ts":"2019-05-12T12:46:03.231Z","test/unit/inheritance.test.ts":"2019-05-12T13:09:16.523Z","test/unit/messages.test.ts":"2019-05-12T12:35:00.602Z","test/unit/nested.test.ts":"2019-05-12T13:07:05.851Z"}}
```

Comment:
🙏 Could we remove this, and Git-ignore it? I'd prefer not to include any editor-specific files. (This also seems specific to a particular extension.)

Reply:
Ah sorry, that was an accidental checkin. I’ve added .vscode to .gitignore in my other PR. I’ll remove the file from this one.